modules/installing-gitops-operator-using-cli.adoc

```diff
@@ -10,7 +10,47 @@ You can install {gitops-title} Operator from the OperatorHub by using the CLI.
 
 .Procedure
```

Comment:
@Srivaralakshmi because of the new installation namespace, this section needed some additional changes. It was not enough to just change the name of the namespace. I tested the procedure to make sure it works. The changes are based on [OCPBUGS-21819](https://issues.redhat.com/browse/OCPBUGS-21819)